pilot/pkg/bootstrap/server.go

```diff
@@ -211,11 +212,13 @@ type Server struct {
 	fileWatcher      filewatcher.FileWatcher
 }
 
+var podNamespaceEnv, _ = env.GetVar("ISTIO_BOOTSTRAP_OVERRIDE", "", "")
```

Comment:
Looks like you are getting the wrong variable here

Reply:
And that's why we do code reviews!

Thanks.